tools/build/src/common/container-helper.ts

```diff
@@ -0,0 +1,73 @@
+/**********************************************************************
+ * Copyright (c) 2020-2021 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ***********************************************************************/
+
+import { CheEditorContainerYaml, CheEditorVolume } from '../editor/che-editors-yaml';
+import { inject, injectable } from 'inversify';
+
+import { CheEditorMetaInfo } from '../editor/che-editors-meta-info';
+import { VolumeMountHelper } from './volume-mount-helper';
+
+export interface Conatainer {
```

Comment:
```suggestion
export interface Container {
```

looks like a typo there

Reply:
fixed